force-app/main/default/lwc/cspTrustedSiteForm/cspTrustedSiteForm.html

```diff
@@ -1,8 +1,8 @@
 <template>
     <lightning-card title="Create Trusted Site" icon-name="standard:invst_profile">
         <div class="slds-p-around_medium">
-            <lightning-input label="Site Name" value={siteName} onchange={handleNameChange} required></lightning-input>
-            <lightning-input label="Site URL" value={siteUrl} onchange={handleUrlChange} type="url" required></lightning-input>
+            <lightning-input label="Site Name" value={siteName} onchange={handleNameChange} required placeholder="e.g. GoogleApi"></lightning-input>
```

Comment:
You can ignore this, but just tossing it in here as a veteran of the forms world.

It's at times helpful to add a `name` attribute to fields because it enables browser autofill, making it pretty easy to click a field and select a value when testing things.

For example:

```suggestion
<lightning-input name="siteName" label="Site Name" value={siteName} onchange={handleNameChange} required placeholder="e.g. GoogleApi"></lightning-input>
```

Would similarly behave like this:

<img width="389" height="201" alt="image" src="https://github.com/user-attachments/assets/39739619-9143-42bd-94e0-9f2390df6030" />